plenum/common/messages/node_messages.py

```diff
@@ -215,6 +219,19 @@ class Reply(MessageBase):
     )
 
 
+class ActionResult(MessageBase):
+    typename = ACTION_RESULT
+    schema = (
+        (f.IDENTIFIER.nm, AnyValueField()),
+        (f.REQ_ID.nm, AnyValueField()),
+        (f.SIG.nm, AnyValueField()),
+        (f.OPERATION.nm, AnyValueField()),
+        (f.PROTOCOL_VERSION.nm, AnyValueField()),
+        (f.IS_SUCCESS.nm, AnyValueField()),
+        (f.MSG.nm, AnyValueField(optional=True, nullable=True))
```

Comment:
The list of keys of `Reply.result` dictionary for an action must be as follows:
- TXN_TYPE
- f.IDENTIFIER
- f.REQ_ID
- f.IS_SUCCESS
- f.MSG